### ISSUE TITLE
Unable to load DLL 'Microsoft.ui.xaml.dll' or one of its dependencies

### ISSUE THREAD

**Issue by GuildOfCalamity** [unoplatform/uno.todo#502]
Upon startup **App.cs** generates the following exception:
**System.DllNotFoundException: 'Unable to load DLL 'Microsoft.ui.xaml.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)'**

## How to reproduce it (as minimally and precisely as possible)

Cloned the ToDo sample and attempted to run on Windows 10. The build process has no issues.

## Environment

Affected platform(s):
- [X] Windows 10

Visual Studio:
- [X] 2022 (version: 17.6.0 Preview 2.0)

**Comment by davidbuckleyni** [comment]
Have the same issue ten years on simple code clones dont work for uno this is a poor advertisement for a platform that meant to be mature.

**Comment by jeromelaban** [comment]
@davidbuckleyni Thanks for the feedback. This is an issue that is unfortunately commonly found in WinAppSDK, and we're working with Microsoft to get this resolved in a more consistent manner in an updated version of the WinAppSDK package.

In the meantime, see this thread: https://github.com/microsoft/microsoft-ui-xaml/issues/6377#issuecomment-1274048235

**Comment by GuildOfCalamity** [comment]
@jeromelaban I have already tried the **package/unpackaged** trick, but after changing the project to unpackaged it just throws another error. I have no problems changing my own WinUI3 projects to unpacked and they work great, but with Uno it does not.

**Comment by davidbuckleyni** [comment]
@nickrandolph that makes no since then its still self containted more or less same as packaged !

**Comment by nickrandolph** [comment]
Not really, since the difference is that you don't install an unpackaged app. The other alternative is to install the runtime, much like you would for a .net framework app.
